packages/mdc-list/mdc-list.scss

```diff
@@ -24,17 +24,17 @@
 // postcss-bem-linter: define list
 
 .mdc-list {
-  @include mdc-typography(subheading2);
+  @include mdc-typography(subtitle1);
   @include mdc-list-item-primary-text-ink-color(text-primary-on-background);
 
   margin: 0;
   padding: 8px 0;
 
   // According to the mocks and stickersheet, the line-height is adjusted to 24px for text content,
-  // same as for subheading1.
+  // same as for body1.
 
   /* @alternate */
-  line-height: map-get(map-get($mdc-typography-styles, subheading1), line-height);
+  line-height: map-get(map-get($mdc-typography-styles, body1), line-height);
```

Comment:
Have we checked whether all of the nitty-gritty of picking specific values out of the typography styles map is still valid in this file?